test/unit/org/apache/cassandra/db/filter/ColumnFilterTest.java

```diff
@@ -518,8 +518,6 @@ else if ("3.11".equals(clusterMinVersion) || (returnStaticContentOnPartitionWith
 
     private void testRoundTrips(ColumnFilter cf)
     {
-        testRoundTrip(cf, MessagingService.VERSION_30);
-        testRoundTrip(cf, MessagingService.VERSION_3014);
```

Comment:
The two signatures of `testRoundTrips` can be merged since we are always using the same MS versions.